kuma/wiki/tests/test_views_document.py

```diff
@@ -63,6 +64,12 @@ def get_content(content_case, data):
 
 @pytest.fixture
 def section_doc(root_doc, wiki_user):
+    """
+    The content in this document's current revision contains multiple HTML
+    elements with an "id" attribute (or "sections"), and also has a length
+    greater than or equal to 200, which meets the compression threshold of
+    the GZipMiddleware.
```

Comment:
Is the 200-character minimum why ``SECTION5`` was added?

Reply:
Yes, I added `SECTION4` solely for that reason.